### ISSUE TITLE
[feature] Workflow Inputs base on Component fields 

### ISSUE THREAD

**Issue by ivicac** [bytechefhq/bytechef#1026]
As a Citizen Developer

I want to choose between options specific to the components used in workflows

So that I can more easily configure workflows when activating them 

Acceptance Criteria
TODO

Definition of Done
TODO

Technical details:
Expose option based properties of components as Input fields

**Comment by kresimir-coko** [comment]
Can we get more details on this @ivicac and is it still relevant?

**Comment by ivicac** [comment]
Yes, it should allow you to define, in the workflow inputs dialog, the options for a particular component and choose a value from them. For example, for the Slack channel, I need to set the Slack channel ID. Right now, I need to enter it manually. I would like to use a select component to choose a channel, instead of manually entering the Slack channel ID.
